indy_node/utils/node_control_tool.py

```diff
@@ -198,31 +201,31 @@ def _backup_name_ext(self, version):
     def _create_backup(self, version):
         logger.debug('Creating backup for {}'.format(version))
         shutil.make_archive(self._backup_name(version),
-                            self.backup_format, self.indy_dir)
+                            self.backup_format, self.backup_target)
 
     def _restore_from_backup(self, version):
         logger.debug('Restoring from backup for {}'.format(version))
         for file_path in self.files_to_preserve:
             try:
-                shutil.copy2(os.path.join(self.indy_dir, file_path),
+                shutil.copy2(os.path.join(self.backup_target, file_path),
                              os.path.join(self.tmp_dir, file_path))
             except IOError as e:
                 logger.warning(
                     'Copying {} failed due to {}'.format(file_path, e))
         shutil.unpack_archive(self._backup_name_ext(
-            version), self.indy_dir, self.backup_format)
+            version), self.backup_target, self.backup_format)
         for file_path in self.files_to_preserve:
             try:
                 shutil.copy2(os.path.join(self.tmp_dir, file_path),
-                             os.path.join(self.indy_dir, file_path))
+                             os.path.join(self.backup_target, file_path))
             except IOError as e:
                 logger.warning(
                     'Copying {} failed due to {}'.format(file_path, e))
         shutil.rmtree(self.tmp_dir, ignore_errors=True)
 
     def _get_backups(self):
-        files = [os.path.join(self.base_dir, file)
-                 for file in os.listdir(self.base_dir)]
+        files = [os.path.join(self.backup_dir, file)
+                 for file in os.listdir(self.backup_dir)]
```

Comment:
file is a built-in function, it's better to use another name for the variable

Reply:
done